app/containers/Wallet.jsx

```diff
@@ -14,7 +14,7 @@ import TicketsPage from "components/views/TicketsPage/TicketsPage";
 import TutorialsPage from "components/views/TutorialsPage/TutorialsPage";
 import GovernancePage from "components/views/GovernancePage/GovernancePage";
 import ProposalDetailsPage from "components/views/ProposalDetailsPage/ProposalDetailsPage";
-import TrezorPage from "components/views/TrezorPage";
+import TrezorPage from "components/views/TrezorPage/TrezorPage";
```

Comment:
you could introduce an `index.js` in `components/views/TrezorPage` to avoid this duplicate name.
- same holds for other views, but we can do the rest in other PR.